src/Compilers/CSharp/Test/Semantic/Semantics/InlineArrayTests.cs

```diff
@@ -540,6 +540,240 @@ struct Buffer
             Assert.Equal(SpecialType.System_Int32, buffer.TryGetInlineArrayElementType().SpecialType);
         }
 
+        [Fact]
+        public void InlineArrayType_19()
+        {
+            var src = @"
+[System.Runtime.CompilerServices.InlineArray((short)10)]
+struct Buffer
+{
+    private int _element0;
+}
+";
```

Comment:
Couldn't something like the following cause a cycle? Is it worth testing?

```cs
[System.Runtime.CompilerServices.InlineArray(C.B[0])]
public struct Buffer
{
    private int _element0;
}

public static class C
{
    public static Buffer B;
}
```

Reply:
> Couldn't something like the following cause a cycle? Is it worth testing?

I do not see anything unique in this scenario. Also, unless I am missing something, ```CycleThroughAttributes_04``` tests a scenario like that.